.env.example

```diff
@@ -1,15 +1,16 @@
-# AMPLITUDE_WRITE_KEY='key_AMPLITUDE_WRITE_KEY'
-ATLASSIAN_CLIENT_ID='key_ATLASSIAN_CLIENT_ID'
-ATLASSIAN_CLIENT_SECRET='key_ATLASSIAN_CLIENT_SECRET'
-AUTH_INTERNAL_DISABLED='false'
-AUTH_GOOGLE_DISABLED='false'
-AUTH_SSO_DISABLED='false'
+# SERVER CONFIG
```

Comment:
Missing vars:
```
NODE_ENV=''
```

Reply:
`NODE_ENV` should be set on the outside, so it is passed to the `node` process as well. We would need to do something like `source .env && node ...` or `dotenv -e node ...`which we're not doing right no.

Reply:
This is interesting as right now we are NOT doing that. `NODE_ENV` is only passed inside the `.env` file. It was done that way previously. 

How would you document that then?

Reply:
You have a point. We should document it how we're using it. If we only set it in the `.env`, then it should be documented here.

Reply:
you're both right!
we should _never_ define NODE_ENV in our .env. This is because it's generally useless!

webpack replaces `process.env.NODE_ENV` with `"production"` whenever we build for prod. 
(https://webpack.js.org/guides/production/). it's _pretty_ intelligent in how it does the replacement. so it'll also replace `process.env['NODE_ENV']`. but it WON'T add it to the env, so things like `env = process.env; env.NODE_ENV` will still be undefined, but that's OK! The whole purpose of NODE_ENV is to allow for tree shaking.

The only place you should see it in our app is a script that is run outside of webpack, e.g. https://github.com/ParabolInc/parabol/blob/92ef9d0b9ab9d4a836b169478fba8d7a98768db5/packages/server/postgres/getPgConfig.ts#L11-L12


That means
NODE_ENV is already d

Reply:
Places where the variable is found, other than in the pm2.config.js or in the docker folder
```
./scripts/migrate.js:  if (process.env.NODE_ENV === 'test') {
./scripts/migrate.js:    console.log('NODE_ENV is test, loading .env.test...')
./packages/server/postgres/getPgConfig.ts:    // this is used outside of webpack transpiled scripts, so checking NODE_ENV is required
./packages/server/postgres/getPgConfig.ts:    max: Number(process.env.POSTGRES_POOL_SIZE) || process.env.NODE_ENV === 'production' ? 20 : 5
./packages/server/database/migrations/20161125141100-addOrgs.js:  if (process.env.NODE_ENV !== 'production') {
./packages/server/database/migrations/20161125141100-addOrgs.js:    console.warn('NODE_ENV is testing. Removing, not migrating prior users.')
./packages/server/database/migrations/20170516165400-backfillInvoices.js:  if (process.env.NODE_ENV !== 'production') {
./packages/server/database/migrations/20170516165400-backfillInvoices.js:    console.warn('NODE_ENV is testing or devving. Not backfilling invoices')
./packages/server/database/getRethinkConfig.ts:    min: process.env.NODE_ENV === 'production' ? 50 : 3,
./packages/server/database/getRethinkConfig.ts:    buffer: process.env.NODE_ENV === 'production' ? 50 : 3
./packages/server/database/getRethinkConfig.ts:  if (process.env.NODE_ENV && flag(process.env.RETHINKDB_SSL)) {
```

Reply:
in dev you really don't want it consuming 20 connections since server restarts, migrations, etc. will quickly eat up your whole pool, especially if the pool isn't drained correctly. All app code is working as intended!

Reply:
I'm removing NOVE_ENV as requested [here](https://github.com/ParabolInc/parabol/pull/9104#discussion_r1393265748). Even though it is used by the application as stated here.